### ISSUE TITLE
OneCycle learning rate scheduler

### ISSUE THREAD

**Issue by MeowZheng** [open-mmlab/mmcv#896]
Hi,

I have a request about OneCycle learning rate scheduler, could you please add a lr hook for it?
ref: pytorch [OneCycleLR](https://pytorch.org/docs/stable/optim.html?highlight=torch%20optim%20lr_scheduler%20onecyclelr#torch.optim.lr_scheduler.OneCycleLR)

Thank you so much for your help.

Best regards,
Meow
